src/utils/wallet.js

```diff
@@ -440,7 +441,7 @@ class Wallet {
         const accountsIds = await getAccountId(publicKey)
 
         if (!accountsIds.length) {
-            throw new Error(`Cannot find matching public key`);
+            throw new WalletError('Cannot find matching public key', 'account.recoverAccount.errorInvalidSeedPhrase', { aaa: 'bbb' })
```

Comment:
@marcinbodnar 
`, { aaa: 'bbb' }` this doesn't look right. Any reason not to pass `publicKey` as in original suggestion?

Reply:
My bad. Fixed.